Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/triangulate_faces.h

```diff
@@ -272,13 +280,17 @@ class Triangulate_modifier
         set_next(h2, h0, pmesh);
 
         Euler::fill_hole(h0, pmesh);
+        visitor(face(h0, pmesh));
       }
     }
+    visitor.done();
     return true;
   }
 
-  bool triangulate_face_with_hole_filling(face_descriptor f, PM& pmesh)
+  bool triangulate_face_with_hole_filling(face_descriptor f, PM& pmesh, Visitor visitor)
```

Comment:
It should be clear in the doc that the visitor will be copied, thus you'll never get any data member if not a reference like type.